src/utils/order.js

```diff
@@ -0,0 +1,54 @@
+export const orderMinority = dataArr => {
+  const descData = dataArr
+    .concat()
+    .sort((a, b) => a.count - b.count)
```

Comment:
I think if you do `.sort((a, b) => b.count - a.count` here then you won't have to do the `.reverse()` afterward. Same with the below instances of `.reverse()` as well.